index.js

```diff
@@ -16,12 +21,29 @@ module.exports = () => {
 			options
 		});
 
+		// Shim onclick event
+		var notification;
+		setTimeout(function () {
+			var onclickOld = notification.onclick;
+			notification.onclick = function () {
+				ipc.send('notification-onclick-shim', notification);
+				if (onclickOld) onclickOld();
+			};
+		}, 1);
+
+		// Apply overrides
+		options = Object.assign({}, options);
+		if (settings.forceSilent) options.silent = true;
+		if (settings.bodyOverride) options.body = settings.bodyOverride;
```

Comment:
About the `bodyOverride`: is it to override the notifications when notifications are handled natively? If so, maybe it could be nice to override the `icon` as well? Hmm

I like the `forceSilent`. Would there be a benefit to killing the notification all together when that happens, as compared to setting the `silent` flag?